### PR TITLE
Update index.md - Add information on the `rcap`, `rex`, `ric`, and `rch` units

### LINKED ISSUE #509: Add more information about the `rcap`, `rex`, `ric`, and `rch` units
### Proposal

I propose to add more information, such as a paragraph, describing the new `rcap`, `rex`, `ric`, and `rch` units to: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units and https://developer.mozilla.org/en-US/docs/Web/CSS/length.

### Browser support

It is supported in Safari 17.2, Opera, Edge and Chrome.
https://developer.apple.com/documentation/safari-release-notes/safari-17_2-release-notes

WebKit commit:
https://github.com/WebKit/WebKit/commit/424dee4815091e03fcc4c9035af6e7a9a8128184

### Tasks

1. Add more information on the `rcap`, `rex`, `ric`, and `rch` units to: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units
2. Add more information on the `rcap`, `rex`, `ric`, and `rch` units to: https://developer.mozilla.org/en-US/docs/Web/CSS/length.

### Dependencies

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by francescorn**
### Description

Adding information on the `rcap`, `rex`, `ric`, and `rch` units.

### Motivation

Anyone can find out more information on the `rcap`, `rex`, `ric`, and `rch` units.

### Additional details

[font-relative-lengths](https://www.w3.org/TR/css-values-4/#font-relative-lengths)

Add editor's spec: [font-relative-lengths](https://drafts.csswg.org/css-values-4/#font-relative-lengths)

### Related issues and pull requests

Fixes mdn/mdn#509

### CODE REVIEW

**Comment by francescorn** [comment]
@estelle @Rumyra @dipikabh Can one of you review my patch when you have a chance? I am fixing my linting errors from now on 😅.

**Comment by dipikabh** [comment]
Thanks @francescorn, I can take a look on Monday

File: files/en-us/web/css/length/index.md [hunk 1]
```diff
@@ -43,11 +43,23 @@ Font lengths define the `<length>` value in terms of the size of a particular ch
...
+- `rex`
+- : Represents the x-height of the root element's {{Cssxref("font")}}.
+- `ric`
+- : Equal to the used {{Glossary("advance measure")}} of the "水" glyph (CJK water ideograph, U+6C34), found of the root element used to render it.
 - `rlh`
-  - : Equal to the computed value of the {{Cssxref("line-height")}} property on the root element (typically {{HTMLElement("html")}}), converted to an absolute length. When used on the {{Cssxref("font-size")}} or {{Cssxref("line-height")}} properties of the root element, it refers to the properties' initial value.
+  - : Equal to the computed value of the {{Cssxref("line-height")}} property on the root element (typically {{HTMLElement("html")}}), converted to an absolute length. When used on the {{Cssxref("font-size")}} or {{Cssxref("line-height")}} properties of the root element, it refers to the properties' initial value. The size of actual lines boxes may differ based on their content as the unit enables length calculations based on the theoretical size of an ideal empty line.
+  -
```
Comment (dipikabh) [inline files/en-us/web/css/length/index.md:62]: The definition list is not rendering properly. Hopefully, this fixes it.
```suggestion
 
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for adding this info, @francescorn!
I've added a few suggestions.

Also fyi, we work from the Editor's draft of specs for MDN updates: https://drafts.csswg.org/css-values-4/#font-relative-lengths

**Comment by francescorn** [comment]
Thank you for letting me know @dipikabh. I will reference the editor's draft specs in future prs. The above changes look good to me.

File: files/en-us/web/css/length/index.md [hunk 2]
```diff
@@ -43,11 +43,22 @@ Font lengths define the `<length>` value in terms of the size of a particular ch
...
+  - : Equal to the "cap height" (nominal height of capital letters) of the root element's {{Cssxref("font")}}.
+- `rch`
+  - : Equal to the width or the {{Glossary("advance measure")}} of the glyph `0` (zero, the Unicode character U+0030) in the root element's {{Cssxref("font")}}.
 - `rem`
-  - : Represents the {{Cssxref("font-size")}} of the root element (typically {{HTMLElement("html")}}). When used within the root element {{Cssxref("font-size")}}, it represents its initial value (a common browser default is `16px`, but user-defined preferences may modify this).
+  - : Represents the {{Cssxref("font-size")}} of the root element (typically {{HTMLElement("html")}}). When used within the root element {{Cssxref("font-size")}}, it represents its initial value. A common browser default is `16px`, but user-defined preferences may modify this.
+- `rex`
+- : Represents the x-height of the root element's {{Cssxref("font")}}.
+- `ric`
+- : Equal to the value of [`ic`](#ic) unit on the root element's font.
```
Comment (dipikabh) [inline files/en-us/web/css/length/index.md:59]:
```suggestion
- `rex`
  - : Represents the x-height of the root element's {{Cssxref("font")}}.
- `ric`
  - : Equal to the value of [`ic`](#ic) unit on the root element's font.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thank you for incorporating the fixes.
There still seems to be some issue with the rendering of the definition list. Please check if the suggested indentations fix the issue.

**Comment by francescorn** [comment]
The tests passed.

**Review by dipikabh — APPROVED** [review]
Perfect, thank you!

**Pull request merged**